test/server/http-test._js

```diff
@@ -29,6 +36,13 @@ asyncTest("Echo service test", 5, (_) => {
     //
     const reader = ez.factory("http://localhost:3004").reader(_);
     strictEqual(reader.read(_), "reply for GET", "Get test: reader ok");
+    // try not found reader
+    try {
+        const nf_reader = ez.factory("http://localhost:3004?status=404").reader(_);
+        ok(false, "Reader supposed to throw");
+    } catch(ex) {
+        ok(/.*Status 404.*/.test(ex.message), "Reader throws ok");
```

Comment:
You can simplify the regex as `/Status 404/`. If you are not anchoring it with `^` and/or '$' it matches anywhere in the string.